### PR TITLE
Bugfix/uselogger instead of console

### PR DESCRIPTION
**Pull request by mbaneryd**

### CODE REVIEW

File: knightbus-schedule/src/KnightBus.Schedule/SchedulingPlugin.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ public SchedulingPlugin(IHostConfiguration configuration)
 
         public async Task StartAsync(CancellationToken cancellationToken)
         {
-            ConsoleWriter.WriteLine($"Starting {nameof(SchedulingPlugin)}");
+            _configuration.Log.LogInformation("Starting {SchedulePluginType}", nameof(SchedulingPlugin));
```
Comment (NiklasArbin) [inline knightbus-schedule/src/KnightBus.Schedule/SchedulingPlugin.cs:25]: I think you should be able to inject the logger instead

**Review by NiklasArbin — COMMENTED** [review]
Do we want to leave to old logger implementation?

File: knightbus-schedule/src/KnightBus.Schedule/SchedulingPlugin.cs (hunk 1 above)
Comment (mbaneryd) [reply knightbus-schedule/src/KnightBus.Schedule/SchedulingPlugin.cs:25]: I we change the ctor for the SchedulingPlugin we need to do a new major version, but that might be fine?

**Review by nazgolgerholm1976 — APPROVED** [review]
LGTM

**Review by albincarnstam — APPROVED** [review]

**Review by NiklasArbin — APPROVED** [review]

**Pull request merged**